recipes/imagemagick/all/patches/fix_ssize_t_undefined.patch

```diff
@@ -0,0 +1,53 @@
+diff --git a/MagickCore/nt-base.h b/MagickCore/nt-base.h
+index 7c7678b59..fbfcb372b 100644
+--- a/MagickCore/nt-base.h
++++ b/MagickCore/nt-base.h
+@@ -61,16 +61,10 @@ extern "C" {
+ #  if !defined(SSIZE_MAX)
+ #    define SSIZE_MAX LLONG_MAX
+ #  endif
+-#  if defined(_MSC_VER)
+-#    define MAGICKCORE_SIZEOF_SSIZE_T 8
+-#  endif
+ #else
+ #  if !defined(SSIZE_MAX)
+ #    define SSIZE_MAX LONG_MAX
+ #  endif
+-#  if defined(_MSC_VER)
+-#    define MAGICKCORE_SIZEOF_SSIZE_T 4
+-#  endif
+ #endif
+ #ifndef S_ISCHR
+ #  define S_ISCHR(m) (((m) & S_IFMT) == S_IFCHR)
+diff --git a/config/config.h.in b/config/config.h.in
+index 7ffe13bb6..e8c18bff3 100644
+--- a/config/config.h.in
++++ b/config/config.h.in
+@@ -1,5 +1,11 @@
+ /* config/config.h.in.  Generated from configure.ac by autoheader.  */
+ 
++#if _WIN64
```

Comment:
Should this an `ifdef` check?

Reply:
both work in this particular case, but sure, `ifdef` is better